node/middlewares/customRoutes.ts

```diff
@@ -0,0 +1,52 @@
+import { getAppsRoutes, getUserRoutes } from '../services/routes'
+
+export async function customRoutes(ctx: Context, next: () => Promise<void>) {
+  const start = process.hrtime()
```

Comment:
```suggestion
  const startTime = process.hrtime()
```
To not have any confusion with javascript start function